app/screens/post_options/post_option.js

```diff
@@ -21,6 +21,7 @@ import link from 'assets/images/post_menu/link.png';
 import pin from 'assets/images/post_menu/pin.png';
 import trash from 'assets/images/post_menu/trash.png';
 import reply from 'assets/images/post_menu/reply.png';
+import bookmark from 'assets/images/post_menu/bookmark.png';
```

Comment:
looking at this, I recently create a Mattermost font ( see #3223 ) that has a bunch of icons (not these ones) so it:
1. performs better
2. less resources
3. can be scalable without any sort of pixelation.

Should we consider adding a ticket so we can replace all of this icons across the mobile app with fonts? I think that would be a very good idea.

Reply:
definitely, should UX own that font, so they can regenerate whenever there is a new icon?

Reply:
0/5 who owns it, maybe something that we can discuss first

Reply:
@enahum Is that something that needs to be changed in this PR now, or can this be done later?

Reply:
definitely later